roles/ceph-rgw/tasks/multisite/destroy.yml

```diff
@@ -3,28 +3,38 @@
   command: radosgw-admin user rm --uid=zone.user
   run_once: true
   failed_when: false
+  register: rgw_delete_the_zone_user
+  changed_when: rgw_delete_the_zone_user.rc is 0
```

Comment:
return code is an int not string so you should use `rgw_delete_the_realm.rc == 0` instead, otherwise it will fail

```console
fatal: [localhost]: FAILED! => {"msg": "The conditional check 'rgw_delete_the_realm.rc is 0' failed. The error was: template error while templating string: expected token 'name', got 'integer'. String: {% if rgw_delete_the_realm.rc is 0 %} True {% else %} False {% endif %}"}

```